LocalExtensions.php

```diff
@@ -237,6 +237,10 @@
 	wfLoadExtension( 'CSS' );
 }
 
+if ( $wmgUseCalendarWikivoyage ) {
+	wfLoadExtension( 'Calendar-Wikivoyage' )
```

Comment:
Please use the original name (https://github.com/wikimedia/mediawiki-extensions-Calendar/blob/master/extension.json#L2) for this one or else it won't work

Reply:
I will note that the mediawiki.org docs for this extension are incorrect
![image](https://user-images.githubusercontent.com/54456226/80392373-aab82480-8874-11ea-88e8-993d4d98b880.png)

Reply:
https://www.mediawiki.org/w/index.php?title=Topic:Vlax350jbs1yysjz&action=history